pepper-apis/src/main/java/org/broadinstitute/ddp/db/dto/ActivityInstanceDto.java

```diff
@@ -25,6 +25,7 @@
     private String sessionId;
     private Long firstCompletedAt;
     private boolean allowUnauthenticated;
+    private int lastVisitedSection;
```

Comment:
With an `int` here the unitialized value I believe will be 0. This might not be what we want if 0 is the first section and no one has ever seen the the activity instance.
Perhaps `Integer` would be better to allow for NULL to be a valid value.
On serialized JSON API, we have unfortunately do not make much use of OPTIONAL properties, so we probably need to allow for this to be serialized as NULL value for activity instances  that have not been looked at all/just created.

Reply:
sound reasonable. @yufengwng what do you think?

Reply:
I tend to prefer to avoid `null` as much as possible. For a newly created activity instance that user hasn't physically "looked" at yet, having a default of 0 seems to make some sense, in my opinion. If the notion of "haven't seen it yet" is something useful to capture, then you'll want it be null-able. My opinion is to start with `int` -- it'll be easier to go from `int` to `Integer` than the other way around.